chore: update README (#885)
logto-io/docs: README.md

<p align="center">
  <a href="https://logto.io" target="_blank" align="center" alt="Logto Logo">
      <img src="https://github.com/logto-io.png" width="100">
  </a>
  <br/>
  <span><i><a href="https://logto.io" target="_blank">Logto</a> is an open-source Auth0 alternative designed for modern apps and SaaS products.</i></span>
</p>

# Logto docs

> Logto docs is the official documentation for Logto. Here, you will find comprehensive guides, API references, and tutorials to help you integrate and use Logto effectively.

Reviewer: ```suggestion
This the official documentation repository for Logto. Here, you will find complete guides, API references, and tutorials to help you integrate and use Logto effectively.
```
